clang/test/BoundsSafety/Frontend/ignored_for_ir.ll

```diff
@@ -0,0 +1,5 @@
+; RUN: %clang -fbounds-safety-experimental -x ir -S %s -o /dev/null 2>&1 | FileCheck %s
+; RUN: %clang_cc1 -fbounds-safety-experimental -x ir -S %s -o /dev/null 2>&1 | FileCheck %s
+
```

Comment:
It might be worth having a small comment here explaining why you expect these diagnostics to not fire.

Reply:
@MaskRay fixed!